src/mmw/mmw/settings/base.py

```diff
@@ -12,6 +12,8 @@
 from os.path import abspath, basename, dirname, join, normpath
 from sys import path
 
+from layer_settings import LAYERS  # NOQA
```

Comment:
Never mind. Removing it reminded me why it was there. We want to be able to access `LAYERS` from where ever settings is imported (including the client-side), but we don't actually need to use it directly in the settings file. This triggers the following PEP8 warning:

`'LAYERS' imported but unused F401`